src/leapfrogai_api/backend/rag/leapfrogai_embeddings.py

```diff
@@ -44,7 +45,10 @@ async def aembed_query(self, text: str) -> list[float]:
 
         return list_of_embeddings[0]
 
-    async def _get_model(self, model_name: str = os.getenv("DEFAULT_EMBEDDINGS_MODEL")):
+    async def _get_model(
+        self,
+        model_name: str = os.getenv("DEFAULT_EMBEDDINGS_MODEL") or "text-embeddings",
```

Comment:
Instead of using `or` we should use the optional param in `os.getenv( ... )`

```suggestion
        model_name: str = os.getenv("DEFAULT_EMBEDDINGS_MODEL", "text-embeddings)
```